util/configv3/load_config.go

```diff
@@ -182,8 +184,20 @@ func removeOldTempConfigFiles() error {
 	}
 
 	for _, oldTempFileName := range oldTempFileNames {
-		err = os.Remove(oldTempFileName)
+		fi, err := os.Lstat(oldTempFileName)
 		if err != nil {
+			// ignore if file doesn't exist anymore due to race conditions if multiple cli commands are running in parallel
+			if errors.Is(err, os.ErrNotExist) {
+				continue
+			}
+			return err
+		}
+		// only delete old orphans which are not caught by the signal handler in WriteConfig
```

Comment:
the condition check here is for skipping. therefore it should say something like: "skip newer files created within 5 minutes"